tests/test_classifier/test_classification_report.py

```diff
@@ -141,9 +146,11 @@ def test_pandas_integration(self):
             }, 'f1': {
                 'unoccupied': 0.9800031994880819,
                 'occupied': 0.9366447034972124
+            }, 'support': {
+                'occupied': 0.22519455252918288,
+                'unoccupied': 0.77480544747081714
             }}
 
-    @pytest.mark.skip(reason="requires random state in quick method")
```

Comment:
Thank you for removing this skip! All the tests pass, possibly because of the increased tolerance; but should we also add the random state here as well?